CHANGELOG.md

```diff
@@ -36,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Placeholder text for Cost Description. [#885](https://github.com/rokwire/events-manager/issues/885)
 - Add isGroupPrivate field for user to select. [#913](https://github.com/rokwire/events-manager/issues/913)
 - Make end date a required field. [#911](https://github.com/rokwire/events-manager/issues/911)
+- Add campus hybrid event support. [#915](https://github.com/rokwire/events-manager/issues/915)
```

Comment:
Could you please fix the CHANGELOG? It looks like "Add campus hybrid event support." was removed from the release 3.3.1 section, which is probably incorrect. Also, the entry for this task needs to be moved to the Unreleased section. Thanks.

Reply:
I may have found the issue with the entry for the "Add campus hybrid event" task. Earlier, it was correctly included under 3.3.1 in the CHANGELOG. Please see the diff commits here (https://github.com/rokwire/events-manager/compare/3.3.0...3.3.1). So, I think you just need to add it back under 3.3.1 and remove it from 3.3.0 in this PR. This entry is also missing in the release notes for 3.3.1 here (https://github.com/rokwire/events-manager/releases/tag/3.3.1). Please make these changes when you get a chance. Thanks.